src/wizard/apple/ios.md

```diff
@@ -5,7 +5,11 @@ support_level: production
 type: language
 ---
 
-We recommend installing the SDK with CocoaPods, but we also support alternate [installation methods](/platforms/apple/install/). To integrate Sentry into your Xcode project, specify it in your _Podfile_:
+We support installing the SDK with [CocoaPods](/platforms/apple/install/cocoapods/), [Swift Package Manager](/platforms/apple/install/swift-package-manager/), and [Carthage](/platforms/apple/install/carthage/).
+
+### CocoaPods
+
+To integrate Sentry into your Xcode project, specify it in your _Podfile_:
```

Comment:
```suggestion
To integrate Sentry into your Xcode project, specify it in your `Podfile`:
```

@lizokm - It looks like we more often use inline code formatting for file names in docs, rather than italics. I like italics, but consistency is more important. What do you think?